### DISCUSSION TITLE
2.0 Feedback

### DISCUSSION THREAD

**Discussion by LukeChannings** [LukeChannings/moviematch#55] [General]
MovieMatch 2.0.0 will close most of the open feature requests in Issues, but isn't stable yet and some parts of the UI are unfinished.

This is a place where you can add feedback on version 2.0.

**Comment by LukeChannings** [comment]
There is no automatic `:latest` tag for the 2.0.0 alphas,

<img width="436" alt="Screen Shot 2021-03-08 at 10 41 42" src="https://user-images.githubusercontent.com/461449/110310868-f2fab400-7ffa-11eb-9311-4d2c3b8c2a02.png">

👆 Set your watch settings like above to get notifications of new alpha releases.

**Comment by ajaywest** [comment]
I have made a config.yaml file based on template provided, but when I open moviematch.exe it just flashes up with the dos box then disappears. Prior to not having a config.yaml, it would just tell me no default  config file found, and not take me to any sort of configurations screen.

Version 2.0.0-alpha.7    apologies if in wrong place

![image](https://user-images.githubusercontent.com/83302600/116285022-a2dfd880-a785-11eb-9d11-e80cf7c14557.png)

**Reply by LukeChannings** [reply]
Did you follow [this template](https://github.com/LukeChannings/moviematch/blob/main/docs/configuration.markdown)?
If it crashes on startup it's probably because the configuration is invalid, or maybe the server URL is invalid?

If you open a Command Prompt and run MovieMatch directly you'll be able to see the error.

Something like this might work:

```batch
cd "C:\Users\ajywest\Downloads"
.\moviematch.exe
```

If you start MovieMatch without a `config.yaml` in the current directory you should be able to create a config file through the UI at [http://localhost:8000](http://localhost:8000). Note: The config UI isn't well tested and looks pretty ugly at the moment.
